t/plugin/plugin.t

```diff
@@ -284,7 +284,7 @@ GET /hello
                     error_response = "OK"
                 }},
             }) do
-                local code, body = t('/apisix/admin/global_rules/1',
+                local code, body = t('/apisix/admin/plugin_configs/1',
```

Comment:
why do we change `global_rules` to `plugin_configs` here?

Reply:
> why do we change `global_rules` to `plugin_configs` here?

The global jwt plugin affects later test cases. Here is just a test schema, it is also possible to use ·`plugin_configs`